ops/utils.js

```diff
@@ -2,7 +2,7 @@ const fs = require("fs");
 
 function versionToNum(version) {
   const strlen = version.length;
-  return Number(version.slice(strlen - 2, strlen));
+  return Number(version.slice(strlen - 1, strlen));
```

Comment:
how about checking the Len of the version first and then decide if to do strlen-1 or strlen-2 ? this will works for both 2 and 1 digits rc versions.